copyright/copyright.js

```diff
@@ -0,0 +1,109 @@
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+/* Copyright (c) 2017 Mobify Research & Development Inc. All rights reserved. */
+/* * *  *  * *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  *  * */
+
+const glob = require('glob')
+const fs = require('fs')
+const path = require('path')
+
+const red = '\x1b[31m'
+const green = '\x1b[32m'
+const yellow = '\x1b[33m'
+const magenta = '\x1b[35m'
+const cyan = '\x1b[36m'
+const blackBG = '\x1b[40m'
+const defaultBG = '\x1b[49m'
+const defaultFG = '\x1b[39m'
+
+const args = process.argv.filter((arg) => {
+    return !/node|copyright/.test(arg)
+})
+
+const copyright = {
+    lintMode: false,
+    langs: {},
+    run() {
+        this.buildSupportedExtensions()
+        const processedGlobs = args.map((folder) => {
+            return new Promise((resolve) => {
+                glob(`${folder}`, (err, file) => {
+                    resolve(file[0])
+                })
+            })
+        })
+
+        Promise.all(processedGlobs).then((files) => {
+            const filesMissingHeader = files.filter((file) => {
```

Comment:
This variable appears to contain filenames that do contain the header, rather than ones that are missing the header, right?